prusti-tests/tests/verify/pass/magic-wands/wand-identity2.rs

```diff
@@ -20,14 +20,14 @@ fn identity_use() {
 }
 
 #[ensures(result.val == v)]    // TODO x.val is illegal, but we crash instead of giving a proper error.
```

Comment:
Does this still crash?

Reply:
We no longer crash, but we give an internal error